pkgs/development/tools/rust/rustup/default.nix

```diff
@@ -96,8 +102,23 @@ rustPlatform.buildRustPackage rec {
     # Note: fish completion script is not supported.
     $out/bin/rustup completions bash cargo > "$out/share/bash-completion/completions/cargo"
     $out/bin/rustup completions zsh cargo >  "$out/share/zsh/site-functions/_cargo"
+
+    # add a wrapper script for ld.lld
+    mkdir -p $out/nix-support
+    substituteAll ${path + "/pkgs/build-support/wrapper-common/utils.bash"} $out/nix-support/utils.bash
```

Comment:
We shouldn't reference `path` in nixpkgs as this will cause to create a copy nixpkgs when evaluating this, which is quite expensive. Please use a relative path here.